dev-infrastructure/modules/logs/kusto/tables/backendContainerLogs.kql

```diff
@@ -0,0 +1,56 @@
+.create-merge table backendContainerLogs (
+  timestamp: datetime,
+  log: dynamic,
+  environment: string,
+  cluster: string,
+  namespace: string,
+  containerName: string,
+  containerImage: string,
+  host: string,
+  labels: string,
+  podName: string,
+  region: string,
+  level: string,
+  operation: string,
+  operationId: string,
+  resourceId: string,
+  internalId: string,
+  clusterId: string,
+  clientRequestId: string,
+  correlationRequestId: string,
+  cloudErrorCode: string,
+  cloudErrorMessage: string,
+  subscriptionId: string,
+  tag: string,
+  unmatched: dynamic
+)
+
+.create-or-alter table backendContainerLogs ingestion json mapping 'ingestionMapping'
+```
+[
+  {"column":"log","Properties":{"path":"$.log.log"}},
+  {"column":"timestamp","Properties":{"path":"$.timestamp"}},
+  {"column":"tag","Properties":{"path":"$.tag"}},
+  {"column":"region","Properties":{"path":"$.log.region"}},
+  {"column":"cluster","Properties":{"path":"$.log.cluster"}},
+  {"column":"namespace","Properties":{"path":"$.log.kubernetes.namespace_name"}},
+  {"column":"containerName", "Properties": {"path": "$log.kubernetes.container_name"}},
+  {"column":"containerImage","Properties":{"path":"$.log.kubernetes.container_image"}},
+  {"column":"host","Properties":{"path":"$.log.kubernetes.host"}},
+  {"column":"labels","Properties":{"path":"$.log.kubernetes.labels"}},
+  {"column":"podName","Properties":{"path":"$.log.kubernetes.pod_name"}},
+  {"column":"environment","Properties":{"path":"$.log.environment"}},
+  {"column":"level","Properties":{"path":"$.level"}},
+  {"column":"operation","Properties":{"path":"$.log.operation"}},
+  {"column":"operationId","Properties":{"path":"$.log.operation_id"}},
+  {"column":"resourceId","Properties":{"path":"$.log.resource_id"}},
+  {"column":"internalId","Properties":{"path":"$.log.internal_id"}},
+  {"column":"clusterId","Properties":{"path":"$.log.cluster_id"}},
+  {"column":"clientRequestId","Properties":{"path":"$.log.client_request_id"}},
+  {"column":"correlationRequestId","Properties":{"path":"$.log.correlation_request_id"}},
```

Comment:
I don't know that 
a) we want to remap the structured fields that users have opted to use in their code (they will expect the logs to look the way they wrote them)
b) we want to allow our components to name things "wrong" - e.g. this is just the `correlationId`, I have never ever heard this called `correlation_request_id`. I imagine we want to allow JOIN across tables given this key, would be nice to get some consistency on it

Reply:
I think this is out of scope for this PR?